packages/offix-datastore/src/utils/logger.ts

```diff
@@ -3,3 +3,9 @@ import debug, { Debugger } from "debug";
 export function createLogger(name: string): Debugger {
   return debug(`datasync:${name}`);
 }
+
+export function enableLogger() {
+  if (window?.localStorage) {
+    window.localStorage.setItem('debug', 'datasync::*');
```

Comment:
```suggestion
    window.localStorage.setItem('debug', 'datasync:*');
```

Reply:
Yep.. I have pushed that by mistake. Have that changed.